src/ui/style/palette.scss

```diff
@@ -62,3 +62,9 @@ $gradient: linear-gradient(270deg, #00b6b7 17.51%, #004073 100%);
 // semantic
 $success: #4ad377;
 $error: #ff3648;
+
+// skeleton
+$skeleton-base-light: darkgrey;
+$skeleton-base-dark: gainsboro;
+$skeleton-animation-light: dimgrey;
```

Comment:
Could you turn color names into hex code colors to keep consistency with others please?

Reply:
@lolottetheclash  Here is the stylelint error that we have that prevent us to use hex code for these colors:
`Expected "#696969" to be "dimgrey" (color-named)` -> [color-named](https://stylelint.io/user-guide/rules/color-named)
Should we deactivate this rule?

Reply:
ah yes I see that the linter reminds us this rule!
for me we should remove it since we only use hexa, do you agree to remove this rule `"color-named": "always-where-possible"` ? @fredericvilcot ?

Reply:
I removed the rule and replaced project colors with hexa in [efcc488](https://github.com/okp4/dataverse-portal/pull/167/commits/efcc488681b0f1e35161155419f5e783f1fb9014) and  [939540c](https://github.com/okp4/dataverse-portal/pull/167/commits/939540c7361a020c4fab516df02c8e434f916e67). I can remove these changes if necessary.